File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        res=0 
        cnt=0
        for num in nums:
            if num==0:
                res=max(res,cnt)
                cnt=0
            else:
                cnt+=1
        return max(res,cnt)

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.cpp
class Solution {
public:
    vector<int> maxSlidingWindow(vector<int>& nums, int k) {
        int l=0;
        vector<int> ans;
        for(int r=0;r<=nums.size()-k; r++){
            int ma=nums[r];
            for(int l=r; l<r+k;l++){
               ma= max(ma, nums[l]);
                
            }
            ans.push_back(ma);
        }
        return ans;
    }
};
